app/experimenter/static/rapid/components/experiments/ExperimentDetails.tsx

```diff
@@ -20,6 +20,12 @@ import { ExperimentStatus } from "experimenter-rapid/types/experiment";
 
 export const POLL_TIMEOUT = 30000;
 
+declare global {
+  interface Window {
+    initSvelte(svelteRoot: HTMLElement | null): void;
```

Comment:
Oh I just noticed if you look at the implementation of this below:

https://github.com/mozilla/experimenter/pull/3419/files#diff-78d5c7f0dd2f373f32fc8310fd33a0b4R6

I'm not sure the root element should be allowed to be `null` and this is also missing the `data` arg, but I think it's not being used yet.  Not worth blocking on, just mentioning.

Reply:
Well, `visualizationRef` gets initialized as `null` and `visualizationRef.current` is still `null` on the first call of `useEffect()` so typescript complains if I don't add the `null` there. I think it's fine though because in `useEffect()`, `initSvelte()` is only called when `visualizationRef.current` is not null.

In terms of `data`, yes that arg is just not being used at the moment